### PR TITLE
DOCS: Rewrite "Install OpenVINO Development Tools" page - port to 22.1

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**
Porting:
https://github.com/openvinotoolkit/openvino/pull/13820/

Details:

Rewrote the introductory section to give more explanation of what OpenVINO Dev Tools is
Added "For Python Developers" and "For C++ Developers" sections in the introduction to explain the difference in the install process
Changed the existing "For Python Developers" section to "Installing OpenVINO Development Tools"
In Step 4, added a Sphinx tab to provide different instructions for Python and C++ users. Python users install using pip install openvino-dev, while C++ install using requirements.txt files. (C++ can alternatively install using pip install openvino-dev, but must be careful to use the right version.)
In Step 5, added a sentence to tell users to visit the Troubleshooting page if they encountered errors
Removed the old "For C++ Developers" section
Changed the "What's Next?" section to add more enticing links, similar to https://github.com/openvinotoolkit/openvino/pull/13219

### CODE REVIEW

File: docs/install_guides/installing-model-dev-tools.md [hunk 1]
```diff
@@ -1,147 +1,205 @@
...
+In both cases, Python 3.6 - 3.9 need be installed on your machine before starting.
 
-If you are a Python developer, you can find the main steps below to install OpenVINO Development Tools. For more details, see <https://pypi.org/project/openvino-dev>.
+> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI.
 
-While installing OpenVINO Development Tools, OpenVINO Runtime will also be installed as a dependency, so you don't need to install OpenVINO Runtime separately.
+## <a name="python-developers"></a>For Python Developers
 
-### Step 1. Set Up Python Virtual Environment
+If you are a Python developer, follow the steps in the <a href="openvino_docs_install_guides_install_dev_tools.html#install-dev-tools">Installing OpenVINO Development Tools</a> section on this page to install it. Installing OpenVINO Development Tools will also install OpenVINO Runtime as a dependency, so you don’t need to install OpenVINO Runtime separately. This option is recommended for new users.
```
Comment (xu-yuan1) [inline docs/install_guides/installing-model-dev-tools.md:19]:
```suggestion
If you are a Python developer, follow the steps in the <a href="o#install-dev-tools">Installing OpenVINO Development Tools</a> section on this page to install it. Installing OpenVINO Development Tools will also install OpenVINO Runtime as a dependency, so you don’t need to install OpenVINO Runtime separately. This option is recommended for new users.
```

File: docs/install_guides/installing-model-dev-tools.md [hunk 2]
```diff
@@ -1,147 +1,205 @@
...
+
+  **Alternative: Install the openvino-dev Package from PyPI**
+  You can also install OpenVINO Development Tools from PyPI using the following command.
+
+   .. important::
+    Make sure to specify the `openvino-dev` version that matches your installed version of OpenVINO Runtime. Otherwise, compatibility errors are likely to occur.
+
+   .. code-block:: sh
+
+      pip install openvino-dev[EXTRAS]==2022.2
```
Comment (xu-yuan1) [inline docs/install_guides/installing-model-dev-tools.md:139]:
```suggestion
      pip install openvino-dev[EXTRAS]==2022.1
```

File: docs/install_guides/installing-model-dev-tools.md [hunk 3]
```diff
@@ -1,147 +1,205 @@
...
+
+  where the EXTRAS parameter specifies one or more deep learning frameworks via these values: caffe, kaldi, mxnet, onnx, pytorch, tensorflow, tensorflow2. Make sure that you install the corresponding frameworks for your models. For example:
+
+   .. code-block:: sh
+
+      pip install openvino-dev[tensorflow2,onnx]==2022.2
```
Comment (xu-yuan1) [inline docs/install_guides/installing-model-dev-tools.md:145]:
```suggestion
      pip install openvino-dev[tensorflow2,onnx]==2022.1
```

File: docs/install_guides/installing-model-dev-tools.md [hunk 4]
```diff
@@ -1,147 +1,205 @@
...
 
-## What's Next?
+## <a name="get-started"></a>What's Next?
+Now that you've installed OpenVINO Runtime, you're ready to run your own machine learning applications! Learn more about how to integrate a model in OpenVINO applications by trying out the following tutorials.
 
-Now you may continue with the following tasks:
+@sphinxdirective
+.. tab:: Get started with Python
+
+   Try the `Python Quick Start Example <https://docs.openvino.ai/2022.2/notebooks/201-vision-monodepth-with-output.html>`_ to estimate depth in a scene using an OpenVINO monodepth model in a Jupyter Notebook inside your web browser.
```
Comment (xu-yuan1) [inline docs/install_guides/installing-model-dev-tools.md:173]:
```suggestion
   Try the `Python Quick Start Example <https://docs.openvino.ai/2022.1/notebooks/201-vision-monodepth-with-output.html>`_ to estimate depth in a scene using an OpenVINO monodepth model in a Jupyter Notebook inside your web browser.
```

File: docs/install_guides/installing-model-dev-tools.md [hunk 5]
```diff
@@ -1,147 +1,205 @@
...
+   
+   .. image:: https://user-images.githubusercontent.com/15709723/127752390-f6aa371f-31b5-4846-84b9-18dd4f662406.gif
+      :width: 400
+
+   Visit the :ref:`Tutorials <notebook tutorials>` page for more Jupyter Notebooks to get you started with OpenVINO, such as:
+   
+   * `OpenVINO Python API Tutorial <https://docs.openvino.ai/2022.2/notebooks/002-openvino-api-with-output.html>`_
```
Comment (xu-yuan1) [inline docs/install_guides/installing-model-dev-tools.md:180]:
```suggestion
   * `OpenVINO Python API Tutorial <https://docs.openvino.ai/2022.1/notebooks/002-openvino-api-with-output.html>`_
```

File: docs/install_guides/installing-model-dev-tools.md [hunk 6]
```diff
@@ -1,147 +1,205 @@
...
+   * `Basic image classification program with Hello Image Classification <https://docs.openvino.ai/2022.2/notebooks/001-hello-world-with-output.html>`_
```
Comment (xu-yuan1) [inline docs/install_guides/installing-model-dev-tools.md:181]:
```suggestion
   * `Basic image classification program with Hello Image Classification <https://docs.openvino.ai/2022.1/notebooks/001-hello-world-with-output.html>`_
```

File: docs/install_guides/installing-model-dev-tools.md [hunk 7]
```diff
@@ -1,147 +1,205 @@
...
+   * `Convert a PyTorch model and use it for image background removal <https://docs.openvino.ai/2022.2/notebooks/205-vision-background-removal-with-output.html>`_
```
Comment (xu-yuan1) [inline docs/install_guides/installing-model-dev-tools.md:182]:
```suggestion
   * `Convert a PyTorch model and use it for image background removal <https://docs.openvino.ai/2022.1/notebooks/205-vision-background-removal-with-output.html>`_
```

File: docs/install_guides/installing-model-dev-tools.md (hunk 1 above)
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-model-dev-tools.md:19]: > 

I do not follow this change, could you elaborate it, please?

Comment (xu-yuan1) [reply docs/install_guides/installing-model-dev-tools.md:19]: openvino_docs_install_guides_install_dev_tools.html is the current file, right? we don't need to add the file name when referencing to an anchor in the same file

Comment (sgolebiewski-intel) [reply docs/install_guides/installing-model-dev-tools.md:19]: Well, normally yes. We will see if it works during the tests ;)

**Review by xu-yuan1 — APPROVED** [review]

**Pull request merged**